tests/jsanalyze_test.js

```diff
@@ -65,16 +82,14 @@ describe('jsanalyze', function () {
 			results.contents.should.eql('var myGlobalMethod = function myGlobalMethod() {return this;};');
 		});
 
-		it('transpile and minify works and transforms async/await when targeting old iOS', function () {
-			const results = jsanalyze.analyzeJs('async function other() { return 1; }; async function first() { const result = await other(); return result + 3; };', { transpile: true, minify: true, targets: { ios: 8 } });
-			results.contents.should.eql('function asyncGeneratorStep(gen,resolve,reject,_next,_throw,key,arg){try{var info=gen[key](arg),value=info.value}catch(error){return void reject(error)}info.done?resolve(value):Promise.resolve(value).then(_next,_throw)}function _asyncToGenerator(fn){return function(){var self=this,args=arguments;return new Promise(function(resolve,reject){function _next(value){asyncGeneratorStep(gen,resolve,reject,_next,_throw,"next",value)}function _throw(err){asyncGeneratorStep(gen,resolve,reject,_next,_throw,"throw",err)}var gen=fn.apply(self,args);_next(void 0)})}}function other(){return _other.apply(this,arguments)}function _other(){return _other=_asyncToGenerator(regeneratorRuntime.mark(function _callee(){return regeneratorRuntime.wrap(function _callee$(_context){for(;1;)switch(_context.prev=_context.next){case 0:return _context.abrupt("return",1);case 1:case"end":return _context.stop();}},_callee)})),_other.apply(this,arguments)};function first(){return _first.apply(this,arguments)}function _first(){return _first=_asyncToGenerator(regeneratorRuntime.mark(function _callee2(){var result;return regeneratorRuntime.wrap(function _callee2$(_context2){for(;1;)switch(_context2.prev=_context2.next){case 0:return _context2.next=2,other();case 2:return result=_context2.sent,_context2.abrupt("return",result+3);case 4:case"end":return _context2.stop();}},_callee2)})),_first.apply(this,arguments)};');
-		});
-
 		it('generates source maps inline into generated js file', function () {
 			const inputJSFile = path.join(__dirname, 'resources/input.js');
 			const contents = fs.readFileSync(inputJSFile, 'utf-8');
-			const expectedSourceMap = fs.readJSONSync(inputJSFile + '.map');
+			var expectedSourceMap = fs.readJSONSync(inputJSFile + '.map');
```

Comment:
Try to get in the habit of using template literals:

```suggestion
			var expectedSourceMap = fs.readJSONSync(`${inputJSFile}.map`);
```

Reply:
:+1: fixed